### PR TITLE
fix(users): Prevent etag change on user login and user logout

### PR DESCRIPTION
**Pull request by ioanpocol**

### CODE REVIEW

**Comment by amagdas** [comment]
What's the actual problem we are trying to solve?

**Comment by petrjasek** [comment]
@ioanpocol it can use some test imo
@amagdas etag is changed on logout so if someone wants to edit a user, asks him to logout, and clicks save - it will fail because of etag change.

**Comment by amagdas** [comment]
Ok, thx for the info.

File: server/apps/preferences.py [hunk 1]
```diff
@@ -155,7 +148,6 @@ def on_fetched_item(self, doc):
         self.enhance_document_with_user_privileges(doc)
 
     def on_update(self, updates, original):
-        # Beware, dragons ahead
```
Comment (amagdas) [inline server/apps/preferences.py]: This was such a nice comment, it's a pity to remove it :)

**Comment by amagdas** [comment]
:+1:

**Comment by mugurrus** [comment]
:+1:

**Pull request merged**